### ISSUE TITLE
[RFC] Structured README.md 

### ISSUE THREAD

**Issue by elicoidal** [OCA/maintainer-tools#269]
Do we have a template for repository README.md? I see a lot of different type for README in the repos and I couldnot find a template.

I am thinking a some guidance with following sections:
1. What is the purpose of the repository
2. How to install the modules
* install Odoo => URL
* install Module => Which way (git pull, pypi, docker?)
3. How to report a bug/issue (=>URL). How to contribute (=> URL)
4. List of addons
5. Translation
6. What is the OCA

The objective is to provide some basic information to the audience arriving from Google search and with not necessary information on Odoo.
The idea would not be to recreate a full How-to in the README but only remind the basics steps to get started.

Must have:
* A README.md template that we can use for manual creation

Nice to have:
* Standard template could be almost fully automated for all repos with commands pregenerated based on the repo URL (like Streisand: https://github.com/jlund/streisand).

**Comment by pedrobaeza** [comment]
Yes, we have. It's in https://github.com/OCA/maintainer-quality-tools/tree/master/sample_files

**Comment by elicoidal** [comment]
Thanks @pedrobaeza 
Still the README template is in my opinion too shallow for common people. It assumes that the targeted audience knows about Odoo and all details to get started.
We should drive the reader at least in the most important steps so that he could get the solution running in a minimum effort.
Again the idea is not to reproduce here a full tutorial but at least the basic steps.

**Comment by sbidoul** [comment]
@elicoidal I agree there is room for improvement in the management of repo README's and addons documentation in general. 

I think we need to separate boilerplate OCA content that must be autogenerated and maintained centrally, from content provided by repo and addons maintainers.

**Comment by dreispt** [comment]
I agree with @sbidoul and suggest creating in this repo a `/tenplate/repository/`.

**Comment by sbidoul** [comment]
Perhaps we should do something similar to #335 for the repo-level README.

**Comment by elicoidal** [comment]
something here: https://odoo-community.org/blog/our-blog-1/post/how-to-install-oca-modules-79
in the future maybe a permanent web page

**Comment by sbidoul** [comment]
@elicoidal I meant having a `DESCRIPTION.rst` at the repo level and having the repo-level `README.rst` generated from that (adding badges, addons table and other boiler plate text automatically).

**Comment by sbidoul** [comment]
@pedrobaeza this is about the repo-level readme, I don't hink this is implemented.

**Comment by pedrobaeza** [comment]
OK, reopening, although I'm not sure it's very relevant apart from the current template...

**Comment by pedrobaeza** [comment]
I prefer to not have addons table, and the rest is put only once. The script that creates new branches makes the rest maintaining the variable part.

**Comment by dreispt** [comment]
This is implemented now.

**Comment by sbidoul** [comment]
Is it?

**Comment by dreispt** [comment]
Sorry, confused repo README with module README.
